bin/smack.py

```diff
@@ -322,7 +403,14 @@ def verify_bpl(args):
   if args.smackd:
     print smackdOutput(verifier_output)
 
+
+
   else:
+    if args.error_witness and result == 'error':
```

Comment:
Please unify this with the `trace_file` stuff below — e.g., by case splitting between `smackJsonToXmlGraph` and `error_trace`.